examples/c++/msm/CMakeLists.txt

```diff
@@ -8,18 +8,17 @@ if (${CMAKE_VERSION} VERSION_LESS "3.24.0")
 else()
     set(CMAKE_CUDA_ARCHITECTURES native) # on 3.24+, on earlier it is ignored, and the target is not passed
 endif ()
-project(icicle LANGUAGES CUDA CXX)
+project(example LANGUAGES CUDA CXX)
 
 set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} --expt-relaxed-constexpr")
 set(CMAKE_CUDA_FLAGS_RELEASE "")
 set(CMAKE_CUDA_FLAGS_DEBUG "${CMAKE_CUDA_FLAGS_DEBUG} -g -G -O0")
-# change the path to your Icicle location
-include_directories("../../../icicle")
+
 add_executable(
   example
   example.cu
 )
-
-find_library(NVML_LIBRARY nvidia-ml PATHS /usr/local/cuda-12.0/targets/x86_64-linux/lib/stubs/ )
-target_link_libraries(example ${NVML_LIBRARY})
+target_include_directories(example PRIVATE "../../../icicle/include")
```

Comment:
Can we avoid this relative path? Maybe use some env variable to tell where icicle is? Ideally should add_subdirectory()

Reply:
The idea was to maybe have pre-built libraries in the future. So add_subdirectory will not necessarily be needed. Actually, there are multiple ways for the user to link with Icicle. But I agree with using variable for Icicle path. Just remember that this is just an example and not the actual app